src/components/Footer.js

```diff
@@ -1,34 +1,67 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const Box = styled.div`
+<link href="https://fonts.googleapis.com/css2?family=Bitter:wght@600&family=Roboto+Mono:wght@500&family=Varela+Round&display=swap" rel="stylesheet"></link>
 
-    padding: 80px 60px;
-    background: rgb(248, 248, 248);
+const Box = styled.div`
+    background: rgb(0,0,0);
     position: absolute;
     bottom: 0;
     width: 100%;
+    display: grid;
+    height: 20vh;
+    grid-template-rows: 50% 10% 40%; 
 `;
 
-const Container = styled.div`
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    margin: 0 auto;
+//I have created the footer using grid which has 3 elements.
+const Box1 = styled.div`
+    width: 100%;
+    text-align: center;
+    align-self: center;
+    font-size: 0.8rem;
+    color: rgb(255,204,17);
 `;
 
+const Box2 = styled.div`
+    width: 100%;
+    text-align: center;
+    background: rgb(255,204,17);

```

Comment:
It would be great if you can have variables for each color defined separately such as :
```footer-bag: rgb(255,204,17)```